### ISSUE TITLE
Can't post a Contact with PaymentTerms Day value set to 0

### ISSUE THREAD

**Issue by simonwaterer** [XeroAPI/Xero-Net#199]
The following code demonstrates the issue:

```
var contact = new Contact
{
	Name = "Sample Contact",
	PaymentTerms = new PaymentTerms
	{
		Sales = new Terms { TermType = PaymentTermType.AfterBillDate, Day = 0 },
		Bills = new Terms { TermType = PaymentTermType.AfterBillDate, Day = 0 },
	}
};

Api.Create(contact);
```

The following ValidationError is returned: `Both Day and Type must be supplied when adding or updating payment terms`	

Now, we are not trying to set payment terms with Day = 0, but we are reading a Contact from Xero API which just happens to have the Day = 0 (which seems to be a legal value in the Xero UI), so this causes an error upon writing it back to the Xero API.

The workaround of course is to set PaymentTerms = null before writing the Contact back to Xero API.

But this behaviour wasn't expected. We expect that if an entity is read from the Xero API then it can be written back (round-tripped) without encountering a Validation Error.

**Comment by MatthewSteeples** [comment]
Line 10 from Terms.cs will need removing. This prevents the 0 from being output as part of the XML so the endpoint sees it as missing.

**Comment by MJMortimer** [comment]
Fixed in #215
